modules/www/manifests/init.pp

```diff
@@ -31,6 +31,13 @@
     require => Package['httpd'],
   }
 
+  file { '/etc/sr':
```

Comment:
Could we please not put this between the `wwwcontent` user and its home directory?
Fully before the user is fine, alternatively perhaps put this just before the `  if $competitor_services {` conditional block?